contentElements/MultiLineInputElement.js

```diff
@@ -80,10 +92,9 @@ MultiLineInputElement.prototype.setPointers = function (entitiesArr) {
         if (this.variable() instanceof GlobalVar) {
             this.setVariableBackRef();
         }
-
     }
+    this.modifier().setPointers(entitiesArr);
     this.questionText().setPointers(entitiesArr);
-    this.recalcHeight();
```

Comment:
why was this recalcHeight removed?

Reply:
I add back the recalcHeight

Reply:
this.recalcHeight(); it will not appear in new commit because after pulling the branch it was automatically updated. So it is not visible in changes but I did check the line 84-88:
MultiLineInputElement.prototype.getAllModifiers = function (modifiersArr) {
    modifiersArr.push(this.modifier());
    this.questionText().getAllModifiers(modifiersArr);
    this.recalcHeight();
};

And it is there

Reply:
you added it in the wrong function.

Reply:
Now should be a correct function